open-company/open-company-web
# Carrot 2 NUX

Card: https://trello.com/c/cWMg18zS

To test:
- onboard a new org
- [x] check all onboard steps reflect the new design (check Trello card)
- once you get to All Posts
- [x] do you see the yellow tooltip above All Posts label?
- click new post
- [x] do you see the abstrac tooltip (dark style, not yellow this time)?
- dismiss it
- publish a post
- [x] do you see the WST tooltip once in the expanded post view?
- dismiss it too
- go back to All Posts
- [x] do you NOT see the yellow tooltip anymore?
- invite another author/admin via email
- accept invitation
- go through invite steps
- [x] do you see the yellow tooltip but no New Post button this time?
- open Cmail
- [x] do you see the summary tooltip
- publish a post
- [x] do you see the WST tooltip?
- dismiss it
